app/controllers/users_controller.rb

```diff
@@ -150,6 +150,7 @@ def login_user(user)
     session[:original_uri] = nil
     set_current_taxonomies(user, {:session => session})
     TopbarSweeper.expire_cache
+    user.post_successful_login
```

Comment:
good catch, I moved this just before push and didn't realize it's now called twice